README.md

```diff
@@ -269,46 +269,87 @@ Depending on which robot data you wish to visualize or operation you wish to run
-    > . ${NVM_DIR}/nvm.sh; nvm use v4.2.2; node ${NODE_DEBUG_OPTION} iCubTelemVizServer.js
-    
-    Now using node v4.2.2 (npm v2.14.7)
-    iCub Telemetry server launched!
-    [INFO] |yarp.os.Port| Port /yarpjs/inertial:i active at tcp://192.168.1.100:10117/
-    [INFO] |yarp.os.impl.PortCoreInputUnit| Receiving input from /icubSim/inertial to /yarpjs/inertial:i using tcp
-    ...
-    [INFO] |yarp.os.Port| Port /yarpjs/sysCmdsGenerator/rpc active at tcp://192.168.1.100:10124/
-    OK
-    Opem-MCT static server process started.
-    ICubTelemetry History hosted at http://localhost:8081/history
-    ICubTelemetry Realtime hosted at ws://localhost:8081/realtime
-    listening on http://localhost:3000
-    [OPEN-MCT STATIC SERVER] stdout:
-    > openmctStaticServer@1.0.0 start <user-home>/dev/yarp-openmct/openmctStaticServer
-    > . ${NVM_DIR}/nvm.sh; nvm use v14.17.0; node server.js
-    
-    
-    [OPEN-MCT STATIC SERVER] stdout: Now using node v14.17.0 (npm v6.14.13)
+3. Open the default comfiguration file `<yarp-openmct-root-folder>/config/default.json`. This JSON formatted file holds the ports configuration, the ports prefix and the servers configuration.
+   <details>
+   <summary>[Servers Default Configuration Parameters]</summary>
+   
+   - The ports configuration `portInConfig` define, for each port, the properties **Yarp name**, **local name** and **port 
+     type** ("bottle", "image").
+   - The ports prefix `robotYarpPortPrefix` (`/icub`, `/icubSim`) is by default defined as a field at the root level of the configuration JSON object and its value is used on the Yarp port names. The `default.json` file has a JSON format but its format was "extended" (via the "pre" processing done in [processedDefault.js](https://github.com/ami-iit/yarp-openmct/blob/8c6a6f5ec7e4b42ee7bad3013de15ad2de1d2c4c/config/processedDefault.js)) to support the use of variables in the JSON literals:
+     ```json
+     {
+         "robotYarpPortPrefix": "/icubSim",
+         ...
+         "portInConfig": {
+             "sens.imu": {
+                 "yarpName": "${this.robotYarpPortPrefix}/inertial",
+                 ...
+             },
+             ...
+         }
+     }
+     ```
+    - `telemVizServer`, `consoleServer` and `openmctStaticServer` set respectively the telemetry data server, the control console server and the OpenMCT static server configuration, providing the port number and host name/address.
+    - `[telemVizServer|consoleServer|openmctStaticServer].port`: The default ports are the most commonly used ones.
+    - `[telemVizServer|consoleServer|openmctStaticServer].host`: If you set it to `localhost`, the server will be reachable only to clients running on the same machine. Actually `localhost` points to `127.0.0.1`, the special internal IP address of the loopback virtual adapter. If you set it to the the machine real IP address, the server can be reached by other machines on the same network.
```

Comment:
What happens if you put the address of a different machine?

Reply:
HaHaaaa! I knew you were gonna ask that! I have no idea. There might be internal mechanisms or shenanigans connecting to a remote machine and launching a server. But I doubt it.

It's something worth checking. At the end, if `localhost` or the machine's self address are the only left choices, then we should only allow those and use a keyword for the machine's self IP, like... `selfIPaddress`, and resolve it in the Node.js/Javascript scripts.

What do you think?

Reply:
> At the end, if localhost or the machine's self address are the only left choices, then we should only allow those and use a keyword for the machine's self IP, like... selfIPaddress, and resolve it in the Node.js/Javascript scripts.

Yeah, I posed the question leaving some room for some fancy mechanism of Node.js I did not know about :smile: . But in general, I think we don't need that feature. If we need to do that, we could resort to ``ssh``. Also resolving automatically the address could be dangerous if you have multiple network cards. Can we check if the address we set is actually the address of the machine, and throw a warning otherwise?

If we leave the possibility to the user to specify the address, it is quite probable he/she will mess up. I simply want to reduce that chance.

Reply:
> But in general, I think we don't need that feature. If we need to do that, we could resort to `ssh`

Yes it's safer to stay simple and only consider using the address(es) of the machine.

> Can we check if the address we set is actually the address of the machine, and throw a warning otherwise?
> If we leave the possibility to the user to specify the address, it is quite probable he/she will mess up. I simply want to reduce that chance.

Agree, I think it's possible, I have to investigate. We get the possible IP addresses and throw a warning if none of them matches the one set by the user.

Reply:
Follow-up in https://github.com/ami-iit/yarp-openmct/issues/77 .